src/ZkFold/Cardano/UPLC/Rollup.hs

```diff
@@ -34,18 +34,19 @@ makeIsDataIndexed ''RollupSetup [('RollupSetup,0)]
 
 data RollupRedeemer =
       UpdateRollup ProofBytes [BuiltinByteString]
```

Comment:
In `UpdateRollup ProofBytes [BuiltinByteString]`, does `[BuiltinByteString]` represent a list of blake2b hashes or arbitrary bytestrings?

Reply:
Yes, the list of hashes, i.e. token names.